src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {
+      person.age
+        && (
+          <p className="Person__age">
+            {`I am ${person.age}`}
+          </p>
+        )
+    }
+
+    <p className="Person__partner">
+      {
+        person.isMarried
+          ? `${person.partnerName} is my ${
```

Comment:
Using 'm' and 'f' to represent genders isn't clear. It would be better to use full words like 'male' and 'female' or define constants MALE = 'm' and FEMALE = 'f' for better readability and to avoid errors.